client/me/purchases/paths.js

```diff
@@ -1,5 +1,17 @@
 function list() {
```

Comment:
While we are updating things, this function could have a better name. Maybe `purchasesRoot`?